run_tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
I've run into cases where `/bin/bash` doesn't actually exist and `/usr/bin/env bash` was more reliable.  I think it was on FreeBSD.

OTH, the contents of `run_tests.sh` could be placed in `Makefile`.